### PR TITLE
add test coverage for cli aws update-server

### PR DESCRIPTION
**Pull request by tonylee80**

### CODE REVIEW

File: packages/cli/src/aws/update-server.test.ts [hunk 1]
```diff
@@ -110,4 +151,10 @@ describe('update-server command', () => {
     await main(medplum, ['node', 'index.js', 'aws', 'update-server', 'incomplete']);
     expect(console.log).toBeCalledWith('ECS Cluster not found');
   });
+
+  test('Update server stack missing service', async () => {
```
Comment (tonylee80) [inline packages/cli/src/aws/update-server.test.ts:155]: test coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Fupdate-server.ts&line=21

Comment (tonylee80) [reply packages/cli/src/aws/update-server.test.ts:155]: also this covers:
https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Futils.ts&line=131

File: packages/cli/src/aws/update-app.test.ts [hunk 2]
```diff
@@ -197,6 +197,65 @@ describe('update-app command', () => {
     expect(console.log).toBeCalledWith('Done');
   });
 
+  test('Update app command without optional configs', async () => {
```
Comment (tonylee80) [inline packages/cli/src/aws/update-app.test.ts:200]: Adding test to cover: https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Fupdate-app.ts&line=39

File: packages/cli/src/aws/update-app.test.ts [hunk 3]
```diff
@@ -176,6 +176,68 @@ describe('update-app command', () => {
...
     );
 
+    // Mock the readdirSync to list files to replace variables
+    (fs.readdirSync as jest.Mock).mockImplementation((folderName) => {
+      if (folderName.endsWith('dist')) {
+        return [{ name: 'js', isDirectory: () => true, isFile: () => false }];
+      }
+      return [
+        { name: 'main.js', isDirectory: () => false, isFile: () => true },
+        { name: 'nonejsfile', isDirectory: () => false, isFile: () => true },
```
Comment (tonylee80) [inline packages/cli/src/aws/update-app.test.ts:186]: adds coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Fupdate-app.ts&line=97

**Review by jamestouri — APPROVED** [review]

**Pull request merged**